src/Extractor.php

```diff
@@ -83,8 +89,15 @@ public function extractData(): void
                 $this->removeEmptyFile($table);
             }
         }
-
         $this->createManifestMetadata($tableNamesForManifest);
+
+        if ($this->config->isIncrementalFetching() && $this->lastRow) {
```

Comment:
Zvykneme zapisovat povodny stav, aj ak by nebol ziaden lastRow (co by sa nemalo stat, ak sa riadky z DB nemazu).

Reply:
přidal jsem to tam včetně testu